revolv/project/models.py

```diff
@@ -426,6 +448,43 @@ def categories(self):
         return [category.title for category in self.category_set.all()]
 
     @property
+    def updates(self):
+        return self.update_set.all()
+
+    def add_update(self, text):
+        update = ProjectUpdate(update_text=text, project=self)
+        update.save()
+
+class ProjectUpdate(models.Model):
+    update_text = models.TextField(
+        'Update text',
+        help_text = "What should the update say?"
+    )
+    
+    date = models.DateField(
+        'Date of update creation',
+        help_text = "What time was your update created?",
+        auto_now_add = True
+    )
+    
+    project = models.ForeignKey(
+        Project,
+        related_name="update"
+    )
+
+    def set_update_text(self, text):
+        self.update_text = text
+
+    @property
+    def month_word(self):
```

Comment:
If this is for rendering in the template, you should use [django.contrib.humanize](https://docs.djangoproject.com/en/1.8/ref/contrib/humanize/) instead